src/events/render-advert-label.ts

```diff
@@ -8,14 +8,6 @@ import fastdom from 'utils/fastdom-promise';
 import crossIcon from '../../static/svg/icon/cross.svg';
 
 const shouldRenderLabel = (adSlotNode: HTMLElement): boolean => {
-	if (
-		adSlotNode.classList.contains('ad-slot--fluid') &&
-		!adSlotNode.classList.contains('ad-slot--interscroller') &&
-		!adSlotNode.classList.contains('ad-slot--article-end')
```

Comment:
Did we mean to remove `ad-slot--article-end` ?

Reply:
I thought in this case the logic for this case was tied into the logic for first condition `adSlotNode.classList.contains('ad-slot--fluid')`?
So I think this can be re added added to return false.

Reply:
Ah so now we don't want to render the label for any fluid?